app/src/main/java/com/infomaniak/mail/ui/newMessage/AiViewModel.kt

```diff
@@ -91,13 +95,24 @@ class AiViewModel @Inject constructor(
                     history += AssistantMessage(aiResult.content)
                     SUCCESS
                 } ?: MISSING_CONTENT
-                error?.code == MAX_SYNTAX_TOKENS_REACHED -> PROMPT_TOO_LONG
+                error?.code == MAX_SYNTAX_TOKENS_REACHED -> {
```

Comment:
```suggestion

                error?.code == MAX_SYNTAX_TOKENS_REACHED -> {
```

Reply:
In fact I think we could extract the first 2 clauses in others functions because it starts to be really hard to read what's going on in this function